### PR TITLE
Add hmf data

### PR DESCRIPTION
**Pull request by MatthieuSchaller**
That's it. Just halo mass functions.

### CODE REVIEW

File: data/HaloMassFunction/conversion/convertBocquet2016.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+output_filename = "Bocquet2016.hdf5"
+output_directory = "../"
+
+if not os.path.exists(output_directory):
+    os.mkdir(output_directory)
+
+data = np.loadtxt("../raw/HMF_Colossus.txt")
+
+M_200 = data[:, 0] * unyt.Solar_Mass
```
Comment (JBorrow) [inline data/HaloMassFunction/conversion/convertBocquet2016.py:21]: These should be changed to be cast to `unyt` arrays as:

```python
M_200 = unyt.unyt_array(data[:, 0], units=unyt.Solar_Mass)
```

which avoids weird issues with conversions between units.

File: data/HaloMassFunction/conversion/convertBocquet2016.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Phi = data[:, 2] * unyt.Mpc ** (-3)
+
+comment = (
+    "Halo mass functions at z=0 from the colossus package (Diemer+18) assuming a 'planck13' cosmology. "
```
Comment (JBorrow) [inline data/HaloMassFunction/conversion/convertBocquet2016.py:25]: This line is too long; a maximum of 88 characters please.

**Review by JBorrow — CHANGES_REQUESTED** [review]
Same comments for both

**Comment by MatthieuSchaller** [comment]
Done.

**Pull request merged**